status-im/status-react
# [BUG#2882] Improved wallet asset selection

relates #2882 

EDIT: Addresses https://github.com/status-im/status-react/issues/2974 bug and https://github.com/status-im/status-react/issues/2975

### Summary:

### Steps to test:
- Open Status
- Navigate wallet tab
- Asset selection should open a new screen with a list of relevant tokens (both for send and request)

status: ready
